File: testdata/jennies/rawtypes/variant_panelcfg_full/PHPRawTypes/src/Types/VariantPanelcfgFull/FieldConfig.php
<?php

namespace Grafana\Foundation\Types\VariantPanelcfgFull;

class FieldConfig implements \JsonSerializable {
    public string $timeseriesFieldConfigOption;

    /**
     * @return array<string, mixed>
     */
    public function jsonSerialize(): array
    {
        $data = [
            "timeseries_field_config_option" => $this->timeseriesFieldConfigOption,
        ];
        return $data;
    }
}

File: testdata/jennies/rawtypes/variant_panelcfg_full/CSharpRawTypes/src/Grafana/Foundation/VariantPanelcfgFull/FieldConfig.cs
namespace Grafana.Foundation.VariantPanelcfgFull;


public class FieldConfig
{
    public string TimeseriesFieldConfigOption;

    public FieldConfig()
    {
        this.TimeseriesFieldConfigOption = "";
    }

    public FieldConfig(string timeseriesFieldConfigOption)
    {
        this.TimeseriesFieldConfigOption = timeseriesFieldConfigOption;
    }
}
